ui/src/common/plugins.ts

```diff
@@ -62,8 +63,12 @@ export class PluginContextImpl implements PluginContext {
   }
 
   registerOnDetailsPanelSelectionChange(
-      onDetailsPanelSelectionChange: (newSelection?: Selection) => void) {
-    this.onDetailsPanelSelectionChange = onDetailsPanelSelectionChange;
+    onDetailsPanelSelectionChange: (newSelection?: Selection) => void) {
+  this.onDetailsPanelSelectionChange = onDetailsPanelSelectionChange;
```

Comment:
Not sure why the formatting changed here but it should be reverted to the way it was. (leading spaces were lost)